tests/integration/all_sets.cfg

```diff
@@ -221,14 +225,15 @@ sets = ["enso_diags"]
 plot_type = "scatter"
 case_id = "TREFHT-response-scatter-start-yrs"
 debug = True
-results_dir = "all_sets_results_test"
+results_dir = "tests/integration/all_sets_results_test"
 seasons = ["ANN"]
 test_start_yr = '2012'
 ref_start_yr = '2012'
 end_yr = '2013'
-test_data_path = 'integration_test_data'
+test_data_path = 'tests/integration/integration_test_data'
 test_file = 'TREFHT_201201_201312.nc'
-reference_data_path = 'integration_test_data'
+reference_data_path = 'tests/integration/integration_test_data'
+
```

Comment:
Are these extra blank lines from the formatter? If not, I think they should be removed.

Reply:
Thanks for the catch, it was most likely from an extra whitespace in my find and replace haha. 
Whitespaces have been reverted.

Reply:
Thanks!